doc/source/examples/multiphysics/tracer-in-static-mixer/tracer-in-static-mixer.rst

```diff
@@ -0,0 +1,295 @@
+======================================
+Tracer in Static Mixer
+======================================
+
+In this example, we extend the :doc:`../../sharp-immersed-boundary/3d-rbf-static-mixer/3d-rbf-static-mixer` example by using a passive tracer to study mixing effects more formally.
+
+We inject two phases through a static mixer to verify the mixing effects of its geometry.
+
++-----------------------------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/static_mixer_stl_casing_arrows.png                                                                      |
+|     :align: center                                                                                                          |
+|     :width: 800                                                                                                             |
+|     :name: Surface grid representation of a helix static mixer with its casing.                                             |
+|                                                                                                                             |
++-----------------------------------------------------------------------------------------------------------------------------+
+
+----------------------------------
+Features
+----------------------------------
+
+- Solver: ``lethe-fluid-sharp``
+- Transient problem
+- Displays the use of the tracer physics
+- Usage of the ``average_velocity_profile`` initial condition to accelerate transient auxiliary physics resolution
+
+
+---------------------------
+Files Used in this Example
+---------------------------
+
+All files mentioned below are located in the example's folder (``examples/multiphysics/tracer-in-static-mixer``).
+
+* Parameter file for solving the flow: ``flow_in_long_mixer.prm``;
+* Parameter file for solving the tracer transport: ``flow_in_long_mixer_only_tracer.prm``;
+* Composite geometry file: ``mixer_long.composite``;
+* RBF geometry file: ``RBF_helix.output``. The extension is ``.output`` because it was named from a `bitpit <https://github.com/optimad/bitpit>`_ perspective. RBF preparation was done in the previous :doc:`../../sharp-immersed-boundary/3d-rbf-static-mixer/3d-rbf-static-mixer` example, based on this surface grid from Thingiverse [#thingiverse]_ under CC BY 4.0;
+* Python post-processing script: ``postprocess_tracer.py``.
```

Comment:
```suggestion
All mentioned files are found in the example's folder (``examples/multiphysics/tracer-in-static-mixer``).

* Parameter file for solving the flow: ``flow_in_long_mixer.prm``;
* Parameter file for solving the tracer transport: ``flow_in_long_mixer_only_tracer.prm``;
* Composite geometry file: ``mixer_long.composite``;
* RBF geometry file: ``RBF_helix.output``. The extension is ``.output`` because it was named from a `bitpit <https://github.com/optimad/bitpit>`_ perspective. RBF preparation was done in the previous :doc:`../../sharp-immersed-boundary/3d-rbf-static-mixer/3d-rbf-static-mixer` example, based on this surface grid from Thingiverse [#thingiverse]_ under CC BY 4.0;
* Python post-processing script: ``postprocess_tracer.py``.
```